thirdparty/assimp/.coveralls.yml

```diff
@@ -0,0 +1,2 @@
+service_name: travis-pro
```

Comment:
We don't need this file, and many others added in this commit. We should only include the files that we actually compile/include + the license/copyright documentation.

Reply:
Please also make this commit log more explicit. It's not adding asset import, it's adding the Assimp library in `thirdparty`. It should mention which version is being added, and it should not include any custom changes, it should be the pristine upstream code (as mentioned above, stripped of all unnecessary files, like all those CI/documentation/CMake/editor config files).

Reply:
The changes to `thirdparty/README.md` and `COPYRIGHT.txt` included in the other commit should be in this one, together with the addition of the Assimp thirdparty code they describe.

Reply:
Please check again. 

The list is now:

* CHANGES
* CONTRIBUTING
* CREDITS
* LICENSE

Reply:
I'd remove `CHANGES` and `CONTRIBUTING` too, if our users want to know how to contribute to Assimp, it's simpler to check the upstream repo.

`CREDITS` and `LICENSE` are good to keep indeed.

Reply:
Some other files to remove:
- `.editorconfig`
- `code/res/assimp.rc`
- `contrib/utf8cpp/doc/ReleaseNotes`
- `contrib/utf8cpp/doc/utf8cpp.html`

And I guess `include/assimp/config.h.in` should likely be renamed to `config.h`, no? Or replaced by the `config.h` generated by CMake in the upstream's buildsystem with the options we use.

Reply:
Apart from that this first commit is OK :+1: